tests/kernel/fp_sharing/float_disable/src/k_float_disable.c

```diff
@@ -14,7 +14,7 @@
  */
```

Comment:
You need to enable the test in .yaml for RISCV

Reply:
I pushed a commit enabling these tests, however there are still a few missing pieces. PR #23265 adds the options to set the -mabi and -march flags. And I'm still waiting for my QEMU patch to be merged that will allow the qemu_riscv32 & qemu_riscv64 platforms to be FP enabled (or new board targets). I've added filters to the yaml for RISC-V so only targets with CONFIG_CPU_HAS_FPU will be included. I don't believe there are any to date...